package.json

```diff
@@ -4,19 +4,22 @@
   "private": true,
   "dependencies": {
     "@firebase/auth-types": "^0.10.0",
-    "@material-ui/core": "^4.9.7",
+    "@material-ui/core": "^4.9.9",
```

Comment:
Yeh none of these packages should need to updated since we're using `^`.
Looks like something may have auto updated these?

Can you revert?